blockcypher/api.py

```diff
@@ -1499,6 +1500,15 @@ def create_unsigned_tx(inputs, outputs, change_address=None,
             inputs_cleaned.append({
                 'addresses': [address, ],
                 })
+        elif 'prev_hash' and  'output_index' in input_obj:
+            prev_hash = input_obj['prev_hash']
+            output_index = input_obj['output_index']
+            assert uses_only_hash_chars(prev_hash), prev_hash
+            assert str(output_index).isdigit(), output_index
```

Comment:
@mflaxman re unit test, yes, no problem. I'm using iOS (Pythonista), so it'll probably be tomorrow
1. My code uses input parameters of format `txid:index`: I find this to be clearest since outputs are also written as `address:value`. 
2. Perhaps [these regexs](https://github.com/wizardofozzie/pybitcointools/blob/master/bitcoin/pyspecials.py#L12-L27) would be worth considering to assert pubkey format